packages/cli/__tests__/zosfiles/__system__/create/ds/cli.files.create.ds.system.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Session } from "@zowe/imperative";
+import { runCliScript, delay, delTime } from "../../../../../../../__tests__/__src__/TestUtils";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { ITestEnvironment } from "../../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { ITestPropertiesSchema } from "../../../../../../../__tests__/__src__/properties/ITestPropertiesSchema";
+import { Delete } from "@zowe/zos-files-for-zowe-sdk";
+
+const ZOWE_OPT_BASE_PATH = "ZOWE_OPT_BASE_PATH";
+
+let REAL_SESSION: Session;
+// Test Environment populated in the beforeAll();
+let TEST_ENVIRONMENT: ITestEnvironment;
+let defaultSystem: ITestPropertiesSchema;
+let dsname: string;
+let dsnameSuffix: string;
+let user: string;
+
+describe("Create Data Set", () => {
+    // Create the unique test environment
+    beforeAll(async () => {
+        TEST_ENVIRONMENT = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
+            testName: "zos_create_dataset"
+        });
+
+        defaultSystem = TEST_ENVIRONMENT.systemTestProperties;
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(TEST_ENVIRONMENT);
+
+        user = defaultSystem.zosmf.user.trim().toUpperCase();
+        dsname = `${user}.TEST.DATA.SET`;
+
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(TEST_ENVIRONMENT);
+    });
+
+    describe("Success scenarios", () => {
+
+        beforeEach(() => {
+            dsnameSuffix = "";  // reset
+        });
+
+        afterEach(async () => {
+            // use DELETE APIs
+            if (dsnameSuffix !== "") {
+                await delay(delTime);
+                const response = await Delete.dataSet(REAL_SESSION, dsname + "." + dsnameSuffix);
+            }
+        });
+
+        it("should create a data set", () => {
+            dsnameSuffix = "";
```

Comment:
Please define a suffix here (for example `dsnameSuffix = "like"`), so that the condition on line 58 will evaluate to true and the data set will be cleaned up after test finishes running.